components/library/tinyxml/Makefile

```diff
@@ -51,3 +51,7 @@ COMPONENT_POST_INSTALL_ACTION= \
 REQUIRED_PACKAGES += system/library
 REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
 REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
+# Auto-generated dependencies
+REQUIRED_PACKAGES += $(GCC_RUNTIME_PKG)
+REQUIRED_PACKAGES += $(GXX_RUNTIME_PKG)
+REQUIRED_PACKAGES += system/library
```

Comment:
Please cleanup duplicates (basically all `REQUIRED_PACKAGES` lines above the `# Auto-generated dependencies` comment line should be removed).

Reply:
re-publish of tinyxml and opal was ok
i did not install/test the resulting packages as its just a rebiuld(?)